openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -198,9 +206,11 @@ This is an example of a Wallet Instance Attestation:
 
 * The Credential Issuer MUST publish a mapping of every Credential Type it supports to a scope value.
 
-# OpenID for Verifiable Presentations
+# OpenID for Verifiable Presentations profile for IETF SD-JWT VC over HTTPS
+
+The requirements for the Wallet and the Verifier, unless specified otherwise:
 
-   * MUST support protocol extensions for SD-JWT VC credential format profile as defined in this specification (#vc_sd_jwt_profile).
+   * MUST support protocol extensions for IETF SD-JWT VC credential format profile as defined in this specification (#vc_sd_jwt_profile).
```

Comment:
```suggestion
   * Wallets and Verifiers MUST support protocol extensions for IETF SD-JWT VC credential format profile as defined in this specification (#vc_sd_jwt_profile).
```

Reply:
Is "protocol extensions" the right wording here? The linked section doesn't use these words.

Reply:
```suggestion
```
i feel like the whole sentence is redundant